### DISCUSSION TITLE
Right-to-Left Support in .NET MAUI Shell TabBar

### DISCUSSION THREAD

**Discussion by uSafi** [dotnet/maui#18578] [Ideas]
.NET MAUI Shell's TabBar currently lacks a FlowDirection property for right-to-left tab order. This feature is crucial for apps targeting right-to-left languages or requiring a right-to-left layout. I propose adding a FlowDirection property to TabBar for better internationalization and design flexibility.
 
 The Problem:
The lack of right-to-left forced me to use Syncfusion's SfTabView, this one is great, looks cool, and has many UI properties one can play with, but it doesn't host ContentPages, only ContentViews and I couldn't find a way around both problems: MAUI TabBar and SfTabView.
So I guess you can feel my frustration.

Please consider this feature for a future update of .NET MAUI.

Thank you for your attention to this matter.

**Comment by asoliman1984** [comment]
Same problem here
